.github/workflows/build-test-measure.yml

```
@@ -0,0 +1,147 @@
+name: Build, test & measure
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    # Run workflow whenever a PR is opened, updated (synchronized), or marked ready for review.
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  # Build a release ZIP file and upload it as a GitHub Actions artifact.
+  release-zip:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    outputs:
+      branch-name: ${{ steps.retrieve-branch-name.outputs.branch_name }}
+      git-sha-8: ${{ steps.retrieve-git-sha-8.outputs.sha8 }}
+
+    steps:
+      - name: Check out source files
+        uses: actions/checkout@v2
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Configure Composer cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        # Scripts are not ignored as they are needed to apply patches for the
+        # `sabberworm/php-css-parser` dependency.
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: Get npm cache directory
+        id: npm-cache
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
+
+      - name: Configure npm cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.npm-cache.outputs.dir }}
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Node dependencies
+        # Prevent malicious scripts from being run with `--ignore-scripts`
+        run: npm install --ignore-scripts
+
+      - name: Create destination directories
+        run: mkdir -p builds/{dev,prod}
+
+      - name: Build develop version
+        run: |
+          npm run build:dev
+          mv amp.zip builds/dev/amp.zip
+
+      - name: Build production version
+        run: |
+          npm run build:prod
+          mv amp.zip builds/prod/amp.zip
+
+      - name: Retrieve branch name
+        id: retrieve-branch-name
+        run: echo "::set-output name=branch_name::$(REF=${GITHUB_HEAD_REF:-$GITHUB_REF} && echo ${REF#refs/heads/} | sed 's/\//-/g')"
+
+      - name: Retrieve git SHA-8 string
+        id: retrieve-git-sha-8
+        run: echo "::set-output name=sha8::$(echo ${GITHUB_SHA} | cut -c1-8)"
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: amp-${{ steps.retrieve-branch-name.outputs.branch_name }}-${{ steps.retrieve-git-sha-8.outputs.sha8 }}
+          path: builds
+
+  deploy-to-gs:
```

Acronym should be GCS for Google Cloud Storage.
```suggestion
  upload-to-gcs:
    name: Upload plugin ZIPs to Google Cloud Storage
```
And, yes, I know that their protocol says `gs`... 😕

---

Good point.